.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+Please make sure to document all user-facing changes in the `CANGELOG.md` file.
+
+Please ensure the optional CI steps do not fail because of this PR. Unless otherwise requested, PRs
+are reviewed once CI succeeeds.
```

Comment:
Since GitHub has draft PRs now, would it be worth mentioning our acceptance of drafts, even in unfinished/broken states? Thoughts about this?

Reply:
> our acceptance of drafts, even in unfinished/broken states

What do you mean by that? Do you mean that we're open to draft PRs? If so, that's a great suggestion.

If you mean that we should tell people that we will merge broken WIP PRs, then I'm a bit confused. :smile:

Reply:
I took it as "we won't automagically close draft/WIP PRs as you work on them."

Reply:
Yea exactly @cole-h, I was just trying to say that we should encourage people to post drafts earlier rather than later, despite the fact they may not work yet, or contain horrible code.